app/api/controllers/validators/resolvers/ResolveTaxYear.scala

```diff
@@ -70,6 +70,19 @@ object ResolveTysTaxYear extends TaxYearResolving {
 
 }
 
+object ResolveTaxYear2026 extends TaxYearResolving {
+
+  def apply(value: String, maybeError: Option[MtdError], errorPath: Option[String]): Validated[Seq[MtdError], TaxYear] =
+    resolve(value, maybeError, errorPath)
+      .andThen { taxYear =>
+        if (taxYear.year < 2026)
+          Invalid(List(InvalidTaxYearParameterError) ++ maybeError)
+        else
+          Valid(taxYear)
+      }
+
+}
+
 case class DetailedResolveTaxYear(
```

Comment:
Could we have used this existing detailed resolver? It can deal with minimum tax year and would give us RULE_TAX_YEAR_NOT_ENDED error as well